### PR TITLE
Split test for showing a home hero banner instead of the explore page…

### PR DESCRIPTION
**Pull request by hworld**
… for guests

### CODE REVIEW

File: src/app/components/shell/footer/footer.vue [hunk 1]
```diff
@@ -10,9 +10,7 @@
 						<br class="visible-xs" />
 						Test the beta version of the Game Jolt app.
 					</p>
-					<p>
-						<app-app-buttons source="footer" />
-					</p>
+					<app-app-buttons source="footer" centered />
```
Comment (hapurubokka) [inline src/app/components/shell/footer/footer.vue:13]: Is `centered` meant to be a prop, or are we assigning styling to it through this somewhere? Is this meant to be using the `justified` prop instead?

Comment (hapurubokka) [reply src/app/components/shell/footer/footer.vue:13]: This also seems to be requiring a `source!: AppPromotionSource` prop.

File: src/app/views/discover/home/_fs-post/fs-post.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+			backgroundImage: `url('${this.mediaItem.mediaserver_url}')`,
+		};
+	}
+
+	mounted() {
+		if (this.video?.postCardVideo) {
+			this.videoController = new VideoPlayerController(this.video.postCardVideo, null);
+
+			this.videoController.volume = 0;
+			this.videoController.muted = true;
```
Comment (hapurubokka) [inline src/app/views/discover/home/_fs-post/fs-post.ts:60]: Would it be preferred to pass in a `VideoPlayerControllerContext` other than `null` here, either using `'gif'` or creating a different one that stays muted?

File: src/app/views/discover/home/_fs-post/fs-post.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		}
+
+		return this.post.videos[0];
+	}
+
+	get bgStyle() {
+		return {
+			backgroundImage: `url('${this.mediaItem.mediaserver_url}')`,
+		};
+	}
```
Comment (hapurubokka) [inline src/app/views/discover/home/_fs-post/fs-post.ts:53]: doesn't appear to be used

File: src/app/views/discover/home/home.ts [hunk 4]
```diff
@@ -33,19 +28,45 @@ import AppDiscoverHomeCommunities from './_communities/communities.vue';
...
+					return;
+				}
+
+				// If we bootstrapped and are logged out, then we've decided to
+				// show this as a split test. If we're logged in, we always use
+				// the default, since it's used as the discover page.
+				if (!this.user) {
+					trackExperimentEngagement(configGuestHome);
+					this.split = configGuestHome.value;
+					this.split = 'hero';
```
Comment (hapurubokka) [inline src/app/views/discover/home/home.ts:60]: This second line should be removed, yeah?

**Review by hapurubokka — CHANGES_REQUESTED** [review]

File: src/app/components/shell/footer/footer.vue (hunk 1 above)
Comment (hworld) [reply src/app/components/shell/footer/footer.vue:13]: good catch, the centered was old. also, source is passed

File: src/app/views/discover/home/home.ts (hunk 4 above)
Comment (hworld) [reply src/app/views/discover/home/home.ts:60]: oh dang, yeah, haha

**Review by hapurubokka — APPROVED** [review]

**Pull request merged**